script/Arcueid/Day 10/10_00_ARC10_5.txt

```diff
@@ -406,7 +406,7 @@ Arcueid claps her hands together in applause.
 -- Page 424, Offset 25339.
 -- 吸血鬼退治に関係のない学校にやってくるなんて、一体なんのつもりなのか。
 // foreshadowiinnggg
-This school has nothing to do with slaying vampires, what is she trying to achieve by coming here?
+This school has nothing to do with slaying vampires, so what is she trying to achieve by coming here?
```

Comment:
Mild pref for without

Reply:
I think it reads more smoothly with the "so" personally